### PR TITLE
Added required attribute on text input for admin controller

### LINKED ISSUE #30046: On AdminImages, when I don't fill a required field, the form is even so saved
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

On images administration, there are some required fields, for example "JPEG compression". 

If I don't fill this field (the field need to be blank), and at the same time, if I update an other field, for example "PNG compression", when I submit the form, I've got an error "JPEG compression field is required" **but** the new value in the PNG compression field is saved. 

### Expected behavior

An error message should be displayed and the other updated fields shouldn't be updated. 

### Steps to reproduce

1. Go to images admin
2. Empty the JPG compression field 
3. Update an other field, for example PNG compression 
4. Submit the form 

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

_No response_

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | On AdminImages, when I don't fill a required field, the form is even so saved
| Type?             | bug fix 
| Category?         | BO
| BC breaks?        |  no
| Deprecations?     | no
| Fixed ticket?     | Fixes #30046
| Related PRs       | 
| How to test?      | See https://github.com/PrestaShop/PrestaShop/issues/30046
| Possible impacts? | 

It's only a front fix. 
There is still no check on server side. 

I only updated the text input part, because the #30046 issue is about this field type, but tell me if we need to update other field types.

### CODE REVIEW

**Review by mflasquin — DISMISSED** [review]

**Review by matks — COMMENTED** [review]
CI did not run 🤔 server problem?

File: admin-dev/themes/default/template/helpers/options/options.tpl [hunk 1]
```diff
@@ -184,7 +184,7 @@
...
 										<div class="col-lg-9">{if isset($field['suffix'])}<div class="input-group{if isset($field.class)} {$field.class}{/if}">{/if}
-											<input class="form-control {if isset($field['class'])}{$field['class']}{/if}" type="{$field['type']}"{if isset($field['id'])} id="{$field['id']}"{/if} size="{if isset($field['size'])}{$field['size']|intval}{else}5{/if}" name="{$key}" value="{if isset($field['no_escape']) && $field['no_escape']}{$field['value']}{else}{$field['value']|escape:'html':'UTF-8'}{/if}" {if isset($field['autocomplete']) && !$field['autocomplete']}autocomplete="off"{/if}/>
+											<input class="form-control {if isset($field['class'])}{$field['class']}{/if}" type="{$field['type']}"{if isset($field['id'])} id="{$field['id']}"{/if} size="{if isset($field['size'])}{$field['size']|intval}{else}5{/if}" name="{$key}" value="{if isset($field['no_escape']) && $field['no_escape']}{$field['value']}{else}{$field['value']|escape:'html':'UTF-8'}{/if}" {if isset($field['autocomplete']) && !$field['autocomplete']}autocomplete="off"{/if} {if isset($field['required']) && $field['required'] && $field['type'] != 'radio'} required="required"{/if}/>
```
Comment (atomiix) [inline admin-dev/themes/default/template/helpers/options/options.tpl]: No need to check the type, we know it's `text` already ;)
```suggestion
											<input class="form-control {if isset($field['class'])}{$field['class']}{/if}" type="{$field['type']}"{if isset($field['id'])} id="{$field['id']}"{/if} size="{if isset($field['size'])}{$field['size']|intval}{else}5{/if}" name="{$key}" value="{if isset($field['no_escape']) && $field['no_escape']}{$field['value']}{else}{$field['value']|escape:'html':'UTF-8'}{/if}" {if isset($field['autocomplete']) && !$field['autocomplete']}autocomplete="off"{/if}{if isset($field['required']) && $field['required'] required="required"{/if}/>
```

Comment (nicosomb) [reply admin-dev/themes/default/template/helpers/options/options.tpl]: Good catch. Fixed.

**Review by matks — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

**Review by MhiriFaten — APPROVED** [review]
Hello @nicosomb ,

I tested your PR and I didn't see the issue, 

[Untitled_ Oct 28, 2022 10_52 AM.webm](https://user-images.githubusercontent.com/111756615/198569965-88f318fe-7277-45dc-8000-e097c20fbd9e.webm)


so it's QA approved :heavy_check_mark: 

Thank you for your PR ! :pray:

**Comment by jolelievre** [comment]
Thanks @nicosomb and @MhiriFaten

**Pull request merged**